lib/widgets/subscription_list.dart

```diff
@@ -41,10 +41,22 @@ class _SubscriptionListPageBodyState extends State<SubscriptionListPageBody> wit
     });
   }
 
+  final _startsWithEmojiRegex = RegExp(r'^\p{Emoji}', unicode: true);
```

Comment:
```
home: Sort leading emoji first in channel names

Update the channel sorting logic to match the channels order
of web by putting emoji-prefixed channels first.

Fixes: #1202
```

Commit-message nit: Putting emoji-prefixed channels first makes our channel-sorting logic more _similar_ to web's, but it still doesn't match completely. Making it match completely is a different issue that we're not ready to complete yet: #1165.

I think you can just remove this sentence from the commit message, leaving:

```
home: Sort leading emoji first in channel names

Fixes: #1202
```

The issue number gives all the needed explanation, I think.